src/lib.rs

```diff
@@ -254,6 +246,21 @@ impl Shape {
         }
     }
 
+    pub fn indented(config: &Config, indent: Indent) -> Shape {
```

Comment:
Could you swap the arg order please, I think that would be more consistent with other functions